packages/zowe-explorer/src/dataset/ZoweDatasetNode.ts

```diff
@@ -112,16 +112,18 @@ export class ZoweDatasetNode extends ZoweTreeNode implements IZoweDatasetTreeNod
     }
 
     public updateStats(item: any): void {
-        if ("m4date" in item) {
+        if ("c4date" in item || "m4date" in item) {
```

Comment:
Not a required change, but would it be worth checking that `c4date` and `m4date` are both in `item` instead of checking for just one?

```suggestion
        if ("c4date" in item && "m4date" in item) {
```

I haven't been able to reproduce it on my end, but if `c4date` is present and `m4date` is missing, it might throw an error when grabbing the modification date/time/seconds from `item`.